Gemfile.lock

```diff
@@ -474,9 +477,9 @@ GEM
       base64
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.4.2)
-      actionpack (>= 5.2)
-      activesupport (>= 5.2)
+    sprockets-rails (3.5.2)
+      actionpack (>= 6.1)
+      activesupport (>= 6.1)
```

Comment:
Good idea to update sprockets-rails to a later 3 version if possible. But I'd prefer it being done in a separate PR.